ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -89,15 +89,31 @@ parseCommand
     ;
     
 kmeansCommand
-    : KMEANS
-    k=integerLiteral
+    : KMEANS (kmeansParameter)? (COMMA kmeansParameter)*
```

Comment:
Does the following command valid?
`kmeans ,distance_type="type"`
Why not KMENAS (kmeansParameter)* ?

Reply:
No, it's not valid, it will return `"No enum constant org.opensearch.ml.common.parameter.KMeansParams.DistanceType.TYPE"`

Antlr parser would be confused by the `|` we use in `kmeansParameter` to make parameters orderless if we define the command like `KMENAS (kmeansParameter)*`

Reply:
Does this help https://github.com/penghuo/os-sql/commit/1e5f9a58de311b531917dda052a0f970ba6bba7b?

Reply:
Thanks, it helps!